backend/src/target.ts

```diff
@@ -109,14 +107,13 @@ class Target {
     });
   }
 
-  sendBalanceTx(from: AddressOrPair, to: AddressOrPair, amount: number): Promise<void> {
-    const fromAddress = (from as KeyringPair).address;
-    const toAddress = (to as KeyringPair).address;
+  sendBalanceTx(from: SignerWithAddress, toAddress: string, amount: number): Promise<void> {
+    const fromAddress = from.address;
     this.logger.info(`Sending balance ${amount} from ${fromAddress} to ${toAddress}`);
     return new Promise((resolve) => {
       this.api.rx.tx.balances
         .transfer(toAddress, amount * Math.pow(10, 12))
-        .signAndSend(from, { nonce: -1 }, Promise.resolve)
+        .signAndSend(from.address, { nonce: -1, signer: from }, Promise.resolve)
```

Comment:
minor, but maybe can use `fromAddress` variable, or remove it completely and just use `from.address` in both places